src/Node.py

```diff
@@ -47,6 +47,12 @@ def __init__(self, name, quorum_set=None, ledger=None, storage=None):
         # self.messages = []
         self.storage = storage if storage is not None else Storage(self)
 
+        self.nomination_state = {'voted': [], 'accepted': [], 'confirmed': []}
```

Comment:
better to reuse the same pattern alike: 

```suggestion
        default_state = {'voted': [], 'accepted': [], 'confirmed': []}
        self.nomination_state = default_state.copy()
        self.balloting_state = default_state.copy()
        
```